src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@ public interface CinemaHallDao {
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
```

Comment:
Consider changing the return type of the `update` method from `boolean` to `boolean` to avoid unnecessary boxing and potential nullability issues.